dist/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;1,500&display=swap');
+    </style>
+    <title>Leaderboard</title>
+  <script defer src="bundle.js"></script></head>
+  <body>
+    <h1 class="page-header">Leaderboard</h1>
+    <section id="leaderboard-scores" class="leaderboard">
+      <div class="display-scores">
+        <div class="display-scores-header">
+          <h2 class="recent-score-title">Recent Scores</h2>
+          <button id="refresh" type="button" class="btn-refresh">
+            Refresh
+          </button>
+        </div>
+        <div class="current-scores">
+          <ul class="score-list">
```

Comment:
- [ ] OPTIONAL: You have done a great job setting up the project with webpack, everything works perfectly! I only recommend to center items in your project. Currently all sections are located on the left side.

Reply:
Thanks for the review @ShoiraTa  the template kind of implies it should be left-aligned ...which was bothering me. 

I will centre everything now. Cheers.